res/controllers/Hercules-P32-scripts.js

```diff
@@ -445,12 +435,7 @@ P32.Deck = function (deckNumbers, channel) {
     };
 
     this.ejectTrack = function (channel, control, value, status, group) {
-        if (value === 127) {
-            engine.setValue(this.currentDeck, 'eject', 1);
-            engine.beginTimer(225, function () {
-                engine.setValue(this.currentDeck, 'eject', 0);
-            }, true);
-        }
+        engine.setValue(this.currentDeck, 'eject', value / 127);
```

Comment:
No, I realized that wasn't appropriate here because there is a button up event... although, thinking about it again, this implementation requires that the Load button is released before the Shift button, which is the problem I solved for Components Buttons in #1271. I think I'll refactor this to use Components.